common/filestore_utils.py

```diff
@@ -13,8 +13,12 @@
 # limitations under the License.
 """Helper functions for interacting with the file storage."""
 
+import functools
+
 from common import experiment_utils
 
+#pylint: disable=invalid-name,global-at-module-level,redefined-outer-name,import-outside-toplevel
```

Comment:
Which of these directives are actually needed?
Is it all of them? To me this looks copied from some of my unittests.
Also, disabling at the toplevel is generally bad. I do it in unittests some times because pylint constantly complains about some things with fixtures. But the better way to do it is to put the pylint directive on the line pylint complains about.

Reply:
They are all needed and not copied from other places.
I can move some one-time use to that line. Thanks.